test/python/dc_test.py

```diff
@@ -0,0 +1,121 @@
+#%%
```

Comment:
Can you please add the default triqs header on top: 
```
##########################################################################
#
# TRIQS: a Toolbox for Research in Interacting Quantum Systems
#
# Copyright (C) 2023 by A. Carta
#
# TRIQS is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later
# version.
#
# TRIQS is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
# details.
#
# You should have received a copy of the GNU General Public License along with
# TRIQS. If not, see <http://www.gnu.org/licenses/>.
#
##########################################################################
```

Reply:
I have fixed the tests after removing the legacy mode, however they now seem quite redundant as they are basically only checking an if statement, I think we need to redesign the tests for them to become more meaningful